From 2241edc8b70c2a54d109ac9c0e821889ac138d40 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Thu, 27 Jan 2022 22:49:52 -0800
Subject: [PATCH] cecloader: Match return type of function LibCecBootloader

Fixes
include/cecloader.h:175:14: error: cannot initialize return object of type 'bool' with an rvalue of type 'nullptr_t'
      return NULL;
             ^~~~

Upstream-Status: Pending
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 include/cecloader.h | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/include/cecloader.h b/include/cecloader.h
index be76468f..550f598e 100644
--- a/include/cecloader.h
+++ b/include/cecloader.h
@@ -172,7 +172,7 @@ bool LibCecBootloader(const char *strLib = NULL)
     if (!g_libCEC)
     {
       std::cout << dlerror() << std::endl;
-      return NULL;
+      return false;
     }
   }
 
@@ -181,7 +181,7 @@ bool LibCecBootloader(const char *strLib = NULL)
   if (!LibCecBootloader)
   {
     std::cout << "cannot find CECStartBootloader" << std::endl;
-    return NULL;
+    return false;
   }
 
   bool bReturn = LibCecBootloader();
-- 
2.35.0

